Sources/Common/Logger/TidalLogger.swift

```diff
@@ -0,0 +1,27 @@
+import Logging
+
+/// Struct abstraction for a logger offering a convenient API for logging.
+public struct TidalLogger {
+	private var logger: Logger
+
+	public init(label: String, level: Logger.Level = .trace) {
```

Comment:
swift-log documentation mentions `.trace` and `.debug` for library developers, so the default is good.